### PR TITLE
ENT-2818 | Enterprise's slug login frontend changes.

### PR DESCRIPTION
**Pull request by hasnain-naveed**
**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

**Comment by hasnain-naveed** [comment]
This is the subtask of ENT-2816. These are the frontend changes but to run it, you should have backend + frontend changes. There is a branch `hasnain-naveed/ENT-2818` which have backend + frontend changes.
@moconnell1453 @saleem-latif @HammadAhmadWaqas @irfanuddinahmad

File: spec/javascripts/fixtures/enterprise_login.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+            <button type="submit" class="background-input enterprise-login-submit-button" id="enterprise-login-submit">
+              Login
+            </button>
+            <span id="activate-progress-icon" class="icon fa fa-spinner fa-spin is-hidden" aria-hidden="true"></span>
+          </form>
+
+      </div>
+    </div>
+  </main>
```
Comment (HammadAhmadWaqas) [inline spec/javascripts/fixtures/enterprise_login.html:31]: new line missing at the end of file

File: enterprise/static/enterprise/js/enterprise_login_page.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            beforeSend: function (xhr) {
+                xhr.setRequestHeader("X-CSRFToken", $.cookie("csrftoken"));
+            },
+
+            data : {
+                enterprise_slug : enterpriseSlug
+            },
+
+            success: function(data) {
+                redirectToURL(data.url);
```
Comment (HammadAhmadWaqas) [inline enterprise/static/enterprise/js/enterprise_login_page.js:26]: Should we display some error to user we will get this URL empty. for example in the case if import error

**Comment by HammadAhmadWaqas** [comment]
can you provide screenshots for the frontend screen?

File: enterprise/static/enterprise/js/enterprise_login_page.js (hunk 2 above)
Comment (hasnain-naveed) [reply enterprise/static/enterprise/js/enterprise_login_page.js:26]: I have handled that case in the backend part. Thanks for this.

File: spec/javascripts/fixtures/enterprise_login.html (hunk 1 above)
Comment (hasnain-naveed) [reply spec/javascripts/fixtures/enterprise_login.html:31]: nope, otherwise I would have got the quality error.

**Comment by hasnain-naveed** [comment]
> can you provide screenshots for the frontend screen?

Screenshot 2020-05-21 at 6 36 39 PM

File: enterprise/static/enterprise/js/enterprise_login_page.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            },
+
+            error : function(xhr) {
+                $("#activate-progress-icon").addClass("is-hidden");
+                $("#enterprise-login-submit").attr("disabled", false);
+                $("#enterprise-login-form-error")
+                .text(xhr.responseJSON.errors.join(", "))
```
Comment (irfanuddinahmad) [inline enterprise/static/enterprise/js/enterprise_login_page.js:33]: Can you show how some of the errors would look to the users?

Comment (hasnain-naveed) [reply enterprise/static/enterprise/js/enterprise_login_page.js:33]: It is same as multiple enterprise selection page.

Screenshot 2020-05-27 at 11 43 27 AM

**Pull request merged**